src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2662,8 +2662,14 @@ limitations under the License.
         <profileId editable="false">0x0103</profileId>
         <deviceId editable="false">0x0077</deviceId>
         <clusters lockOthers="true">
-            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+            <include cluster="On/Off" client="false" server="true" clientLocked="true" serverLocked="true">
```

Comment:
Why is this `serverLocked="true"`?

Reply:
Thank you for pointing this out. serverLocked should be false here because On/Off cluster in Cook Surface device type is optional and serverLocked indicates whether a cluster is mandatory on the server side. I will fix this.